docs/authorization/gare.rst

```diff
@@ -118,22 +115,38 @@ Requirements Error, you can use, e.g.,:
 
 .. code-block:: python
 
-    auth_requirements_error.utils.to_auth_requirements_error(
-        other_error
-    )  # GlobusAuthRequirementsError
-    auth_requirements_error.utils.to_auth_requirements_errors(
-        [other_error]
-    )  # [GlobusAuthRequirementsError, ...]
+    gare.to_auth_requirements_error(other_error)  # GARE
+    gare.to_auth_requirements_errors([other_error])  # [GARE, ...]
```

Comment:
Given that we've gotten ride of the "Auth Requirements Error" name, I have a fairly strong preference that these become `to_gare` and `to_gares`, otherwise I think we're potentially introducing some inconsistency/confusion here.

I think maybe we alias the old method names, perhaps with a v4 deprecation warning?

Reply:
Yeah, I can include that rename and agree with that rationale. I'll come back to this later today.